front/app/containers/Admin/sideBar/index.tsx

```diff
@@ -251,15 +251,15 @@ class Sidebar extends PureComponent<
             ),
         },
         {
-          name: 'emails',
-          link: '/admin/emails',
+          name: 'messaging',
+          link: '/admin/messaging',
           iconName: 'emails',
-          message: 'emails',
+          message: 'messaging',
           isActive: (pathName) =>
             pathName.startsWith(
               `${
                 getUrlLocale(pathName) ? `/${getUrlLocale(pathName)}` : ''
-              }/admin/emails`
+              }/admin/messaging`
             ),
         },
         {
```

Comment:
Are we planning to use the email naming again later? Is messaging just for the index page?

Reply:
Yeah, this is just the index, so `/admin/messaging` which will contain emails and texting once we're done. The previous routes were `/emails/custom` and `/emails/automated` and will now be `messaging/emails/custom` and `messaging/emails/automated`